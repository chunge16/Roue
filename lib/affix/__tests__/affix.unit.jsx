import * as renderer from 'react-test-renderer';
import * as React from 'react';
import {mount} from 'enzyme';
import Affix from '../affix';

describe("Affix", () => {
    it("affix exist", () => {
        const json = renderer.create(<Affix/>).toJSON();
        expect(json).toMatchSnapshot();
    });
    it("affix content", () => {
        const component = mount(<Affix>12345</Affix>);
        expect(component.find(".roue-affix").text()).toEqual("12345");
    });
  /*  it("affix fixed", () => {
        const component = mount(<Affix>12345</Affix>);
        Object.defineProperty(window, 'scrollY', {value: 9990, writable: true})
        console.log(component.find(".roue-affix").props())
        component.find(".roue-affix").prop('onScrollY')
        expect(component.find(".roue-affix").prop('style')).toHaveProperty('position', 'fixed')
    });*/
});
