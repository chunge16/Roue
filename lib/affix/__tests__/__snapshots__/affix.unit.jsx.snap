// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Affix affix exist 1`] = `
<div
  className="roue-affix"
/>
`;
